runtime/compiler/control/CompilationThread.cpp

```diff
@@ -6896,6 +6896,9 @@ TR::CompilationInfoPerThreadBase::shouldPerformLocalComp(const TR_MethodToBeComp
       (!JITServerHelpers::isServerAvailable() && !JITServerHelpers::shouldRetryConnection(OMRPORT_FROM_J9PORT(_jitConfig->javaVM->portLibrary))))
       doLocalComp = true;
 
+   if (TR::Compiler->target.cpu.isPower() && _jitConfig->inlineFieldWatches)
```

Comment:
```suggestion
   // Do a local compile because the Power codegen is missing some FieldWatch relocation support.
   if (TR::Compiler->target.cpu.isPower() && _jitConfig->inlineFieldWatches)
```